snippets/rust.json

```diff
@@ -22,12 +245,12 @@
         ],
         "description": "Insert unreachable!"
     },
-    "println": {
```

Comment:
This macro wasn't removed, but grouped with others such as `write` and `format`.